src/custom/pages/Profile/index.tsx

```diff
@@ -34,7 +36,13 @@ export default function Profile() {
               <RefreshCcw size={16} />
               &nbsp;&nbsp;
               <Txt secondary>Last updated:&nbsp;</Txt>
-              <strong>{lastUpdated || '-'}</strong>
+              {!lastUpdated ? (
+                '-'
+              ) : (
+                <MouseoverTooltipContent content={<TimeFormatted date={profileData?.lastUpdated} />}>
```

Comment:
You missed one tooltip from the issue:

https://github.com/gnosis/cowswap/issues/1567
![screenshot_2021-10-18_16-50-56](https://user-images.githubusercontent.com/43217/137821559-be74f0f0-8e4c-4736-b11f-645f5442c86b.png)

Reply:
Also, the tooltip seems too big for the message contents.

Can we allow it to match the contents? Or is there a minimum width set?